app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ReviewsController < ApplicationController
+  before_action :authenticate_user!
+  
+  def index
+
+  end
+
+  def new
+    @site = Site.find(params[:site_id])
+    @review = Review.new
+  end
+
+  def create
+    @site = Site.find(params[:site_id])
+    @review = Review.new(review_params)
+    @review.site = @site
+    if @review.save
+      redirect_to site_path(@review.site_id), notice: "Review Saved Successfully"
+    else
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.